packages/protocol-parser/src/getKeySchema.ts

```diff
@@ -1,11 +1,13 @@
-import { Schema, Table } from "@latticexyz/config";
+import { KeySchema, StaticAbiType, Table } from "@latticexyz/config";
```

Comment:
need to bring over `KeySchema` into this PR

but I don't think it belongs in the config package, because config has no notion of key schema/value schema, so I think should prob bring that over here instead